drivers/hmis/app/models/hmis/hud/validators/enrollment_validator.rb

```diff
@@ -44,7 +60,30 @@ def self.validate_entry_date(enrollment, household_members: nil, options: {})
 
   def self.hmis_validate(record, role: nil, **_)
     errors = HmisErrors::Errors.new
-    errors.push(*validate_entry_date(record)) if role == :INTAKE
+    errors.push(*validate_entry_date(record)) if [:INTAKE, :ENROLLMENT].include?(role&.to_sym)
+
+    if record.new_record?
+      household_members = record.household_members
+      is_hoh = record.head_of_household?
+      has_hoh = household_members.find(&:head_of_household?).present?
```

Comment:
maybe this is a little clearer
```suggestion
      has_hoh = household_members.any?(&:head_of_household?)
```